### PR TITLE
Run on Elixir 1.11

### PR DESCRIPTION
**Pull request by angelikatyborska**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,11 +7,6 @@ RUN apt-get update && \
 # Create appuser
 RUN useradd -ms /bin/bash appuser
 
-# Get exercism's tooling_webserver
-RUN curl -L -o /usr/local/bin/tooling_webserver \
-  https://github.com/exercism/tooling-webserver/releases/download/0.10.0/tooling_webserver && \
-  chmod +x /usr/local/bin/tooling_webserver
```
Comment (angelikatyborska) [inline Dockerfile:13]: @iHiD Is it safe to remove this from the test runner?

**Comment by neenjaw** [comment]
Why is it failing CI on this branch?

**Comment by angelikatyborska** [comment]
For some reason, using the test runner on `test/hello-world` doesn't produce a `results.json` file anymore. I can't figure out why.

**Comment by neenjaw** [comment]
strange, I'll pull it local to check it out

File: Dockerfile (hunk 1 above)
Comment (iHiD) [reply Dockerfile:13]: Yes

**Comment by neenjaw** [comment]
It runs again now 🎉

**Review by neenjaw — APPROVED** [review]

**Pull request merged**